src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html

```diff
@@ -85,7 +85,16 @@ <h4 *ngIf="isExamMode" class="card-title">{{ 'artemisApp.assessmentDashboard.inf
                             <b>{{ 'artemisApp.assessmentDashboard.totalLocks' | translate }}:</b> {{ totalNumberOfAssessmentLocks }}
                         </a>
                     </li>
-                    <li *ngIf="!isExamMode">
+                    <li *ngIf="!isExamMode && isAtLeastInstructor">
+                        <b>{{ 'artemisApp.assessmentDashboard.totalLocks' | translate }}:</b> {{ totalNumberOfAssessmentLocks }}
+                    </li>
+                    <!-- Complaints about all assessments: -->
+                    <li *ngIf="!isExamMode && complaintsEnabled">
+                        <a [routerLink]="['/course-management', courseId, 'complaints']">
+                            <b> {{ 'artemisApp.assessmentDashboard.totalComplaints' | translate }}:</b> {{ numberOfTutorComplaints }}
+                        </a>
+                    </li>
+                    <li *ngIf="!isExamMode && feedbackRequestEnabled">
```

Comment:
[W3schools](https://www.w3schools.com/tags/tag_b.asp)
`Note: According to the HTML5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate. The specification states that headings should be denoted with the <h1> to <h6> tags, emphasized text should be denoted with the <em> tag, important text should be denoted with the <strong> tag, and marked/highlighted text should be denoted with the <mark> tag.`

Reply:
Oh, I didn't know that, I changed it :+1: